src/components/surveys/survey-design/question-edit/phone-subcontrols/Completion.test.tsx

```diff
@@ -25,6 +25,11 @@ const step: Step = {
   },
 }
 
+afterEach(() => {
+  jest.runOnlyPendingTimers()
+  jest.useRealTimers()
+})
```

Comment:
`renderSurveyQuestionComponent`, which is used to render the survey components, calls `jest.useFakeTimers()`. Real timers should be restored after each test